### DISCUSSION TITLE
Buyer app initiation using search

### DISCUSSION THREAD

**Discussion by nemishzalavadiya-pirimid** [ONDC-Official/ONDC-FIS-Specifications#29] [Q&A] [answered]
Swagger [link](https://ondc-official.github.io/ONDC-FIS-Specifications/#/The%20consumer%20initiates%20the%20personal%20loan%20request%20with%20account%20details.)

According to swagger there are 2 **/search** references 
1. Provider Platform
2. Gateway

Note: implementation seems same for both ref.

So just to clarify Buyer app will call gateway **/search**?

**Comment by Antriksh-Parmar** [comment, marked as answer]
That's correct.
